db/migrate/20170210163241_recents_index_updates.rb

```diff
@@ -0,0 +1,12 @@
+class RecentsIndexUpdates < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def change
+    %i(classification_id project_id subject_id user_id workflow_id).each do |col|
+      remove_index :recents, column: col
+      add_index :recents, col, algorithm: :concurrently
```

Comment:
It rebuilds the index to remove the bloat from deleted removed rows. Currently ~1GB but should be around 700MB